test/js-native-api/test_string/test_string.c

```diff
@@ -221,6 +236,14 @@ static napi_value TestUtf16External(napi_env env, napi_callback_info info) {
                          actual_length);
 }
 
+static napi_value TestPropertyKeyUtf16(napi_env env, napi_callback_info info) {
+  return TestOneByteImpl(env,
+                         info,
+                         napi_get_value_string_utf16,
+                         create_property_key_utf16,
```

Comment:
`create_property_key_utf16` is nothing more than a thin wrapper around `napi_create_string_utf16`. I believe this can be replaced with `napi_create_string_utf16`.

```suggestion
                         napi_create_string_utf16,
```

Reply:
@legendecas , please note that the intent here is to test the new `node_api_create_property_key_utf16`. We should rather change the `create_property_key_utf16` body or use the `node_api_create_property_key_utf16`.

Reply:
Oh, yes, sorry. Updated my suggestion!